packages/cli/src/commands/autocomplete/create.ts

```diff
@@ -74,9 +78,11 @@ export default class Create extends AutocompleteBase {
         try {
           commands.push(c)
         } catch (error: any) {
-          debug(`Error creating completions for command ${c.id}`)
-          debug(error.message)
-          this.writeLogFile(error.message)
+          debugLog(`Error creating completions for command ${c.id}`)
+          debugLog(error.message)
+          this.writeLogFile(error.message).catch(error_ => {
+            debugLog(`Failed to write log file: ${error_.message}`)
+          })
```

Comment:
Is this change necessary? I'm not sure I understand the switch from `error` to `error_`.